jsonchecker/__init__.py

```diff
@@ -42,7 +42,8 @@ def check_directory(self, directory):
         else:
             files = [directory]
         for fname in files:
-            if fname.startswith('.'):
+            first_directory = fname.split(os.path.sep)[0]
```

Comment:
Good point.

So if the last directory in a path is `.` or `..` or begins with `.`, then skip it. Correct?

And could that be generalised to: if the last directory in a path begins with `.`, then skip it?